Doc/reference/datamodel.rst

```diff
@@ -507,7 +507,7 @@ Callable types
       |                         | arbitrary function            |           |
       |                         | attributes.                   |           |
       +-------------------------+-------------------------------+-----------+
-      | :attr:`__closure__`     | ``None`` or a tuple of cells  | Read-only |
+      | :attr:`__closure__`     | ``None`` or a tuple of cells  | Writable  |
```

Comment:
The `__closure__` attribute is not writable. The undocumented `cell_contents` attribute of `__closure__` elements is writable.